### DISCUSSION TITLE
Unable to get bridge working

### DISCUSSION THREAD

**Discussion by haberlj** [isalkind/homebridge-away-mode#27] [Q&A] [answered]
Hello.
I have installed 1.6.5 on HOOBS 4.3.0. I was trying to replicate the simple config example, but the bridge does not show as active. I have 3 other bridge/plugins configured which are working fine. The only thing the log shows (even with debugging enabled) is:
`10/7/2023, 2:20:01 PMawaymodebridge stopping
10/7/2023, 2:20:06 PMAway Mode Bridge starting
10/7/2023, 2:20:09 PMAway Mode BridgeLoaded plugin 'homebridge-away-mode'
10/7/2023, 2:20:10 PMAway Mode BridgeLoading 1 accessories...
10/7/2023, 2:20:10 PMAway Mode BridgeAway ModeSwitch: Away Mode
10/7/2023, 2:20:10 PMAway Mode BridgeAway ModeTrigger 1 (0): Timer On: 15-16s, Off: 30-31s
10/7/2023, 2:20:10 PMAway Mode BridgeWARNINGunhandled rejection: Error`

The config i specified is:
`{
    "accessories": [
        {
            "name": "Away Mode",
            "sensors": [
                {
                    "name": "Trigger 1",
                    "activeTimesForSensor": [
                        {
                            "absolute": true
                        }
                    ]
                }
            ],
            "accessory": "AwayMode",
            "plugin_map": {
                "plugin_name": "homebridge-away-mode"
            },
            "minOffTime": 30,
            "maxOffTime": 31,
            "minOnTime": 15,
            "maxOnTime": 16
        }
    ]
}`

Thanks.
Joe

**Comment by isalkind** [comment, marked as answer]
The config you provided isn't sufficient to run the plugin:
- There's no specification for when the sensors should be active (start/end)
- The `activeTimesForSensor` on Trigger 1 is not needed if you're using globals
- I don't know what the `plugin_map` is for

I've included a simple example of a functional config based on your example:

`{
  "accessories": [
    {
      "name": "Away Mode",
      "sensors": [
        {
          "name": "Trigger 1"
        }
      ],
      "minOffTime": 30,
      "maxOffTime": 31,
      "minOnTime": 15,
      "maxOnTime": 16,
      "activeTimes": [
        {
          "start": "20:00",
          "end": "23:30"
        }
      ],
      "accessory": "AwayMode"
    }
  ]
}`

**Reply by haberlj** [reply]
Thank you. Working baseline. On my way!
